apps/vast/README.md

```diff
@@ -39,4 +39,87 @@ Request an intelligence snapshot of the past 50 days.
 
 ```sh
 ./vast-bridge.py --snapshot=50
-```
\ No newline at end of file
+```
+
+## Bridge Features
+
+This section explains the most important features of the `vast-bridge` and the
+CLI toggles to enable them.
+
+### IoC Matching
+
+VAST can match IoCs either live (pro-feature) or retrospectively via the means
+of usual queries.
+
+#### Live Matching
+
+VAST's
+[live matching](https://docs.tenzir.com/vast/features/threat-intel-matching)
+works as
+[continuous query](https://docs.tenzir.com/vast/cli/vast/export/#documentation).
+The `vast-bridge` subscribes to those continuous query results and reports all
+new IoC matches from VAST to Threat Bus as `Sightings`.
+
+The live matching is the default mode of operation for the `vast-bridge`. Note
+that the VAST node must support this feature.
+
+#### Retro Matching
+
+The `vast-bridge` supports retro matching via the command line toggle
+`--retro-match`. When this toggle is used, the bridge transforms IoCs from
+Threat Bus to valid VAST queries, instead of feeding the IoCs to a live matcher.
+
+Each result from an IoC query is treated as `Sighting` of that IoC and reported
+back to Threat Bus.
+
+### Sighting Context Transformation
+
+The bridge provides a command line option to invoke another program for parsing
+Sighting context data.
+
+A `Sighting` object is structured as follows:
+
+```yaml
+{
+  "ts": "2020-09-04 11:21:13.663164",
+  "reference": "threatbus__86",
+  "ioc": "foobar.com",
+  "context": {
+    ...
+  }
+```
+
+The `context` field can contain arbitrary data. For example, retro matches from
+VAST contain the full query result in the context field (like a Suricata EVE
+entry or a Zeek conn.log entry).
+
+Use the CLI toggle `--transform-context "cmd args" to transform the `context`
+object via some command line tool. For example, pass the `context` object to
+[DCSO/fever](https://github.com/DCSO/fever) `alertify`:
+
+```
+apps/vast/vast-bridge.py --retro-match --transform-context "fever alertify --alert-prefix VAST-RETRO --extra-key vast-ioc --ioc %ioc"
+```
+
+Note that the `cmd` string passed to `--transform-context` is treated as
+template string. The placeholder `%ioc` is replaced with the contents of the
+actually matched IoC.
+
+### Custom Sinks for Sightings
+
+The `vast-bridge` offers to send Sighting `context` to a configurable `sink`
+_instead_ of reporting them back to Threat Bus. This can be configured via the
+`--sink` parameter. The special placeholder `STDOUT` can be used to print the
+Sighting context to `STDOUT`. Example:
+
+```
+apps/vast/vast-bridge.py --sink stdout
+```
+
+A custom sink is useful to forward `Sightings` to another process, like
+`syslog`, or forward STDOUT via a UNIX pipe. Note that it may be desirable to
+disable logging in that case (`--loglevel NOTSET`).
```

Comment:
It would be possible, but I'm against it for two reasons. First, we already have an option to modify the loglevel and second, this is not a UNIX "tool" but rather an application.